locksmith/src/operations/lockOperations.ts

```diff
@@ -74,3 +75,139 @@ export async function getKeyHolderMetadata(
     },
   })
 }
+
+export async function getIcon(address: string) {
```

Comment:
Lets move this and  associated code into its own library, it can still be called from the Lock Controller.  We have been using the "operations" code to separate persistence code from the controller but this does quite fit that paradigm